packages/extract-react-types/src/index.js

```diff
@@ -681,22 +681,14 @@ converters.Identifier = (path, context): K.Id => {
           ? convert(path.get('typeAnnotation'), { ...context, mode: 'type' })
           : null;
 
-        return {
```

Comment:
what's the prettier change tho?

Reply:
oh just some clean up. Totally personal preference here.